graph/README.md

```diff
@@ -0,0 +1,5 @@
+### Instructions
+
+Run `make create-config`. This will create a `config.json` file which the user will need to update with two pieces of information, the part-id of the machine as well as the unix timestamp of the data they wish to graph.
```

Comment:
This instruction will only work on the machine that has run the module.

Reply:
not necessarily,
as long as I have waypoint and trajectory csvs this will work.